gen_cert.py

```diff
@@ -81,10 +81,14 @@ def get_cert_date(
     """
     Get pertinent date for display on cert
     """
-    if force_date:
-        date_value = format_datetime(force_date, 'MMMM d, y', tzinfo=timezone, locale=locale)
-    else:
-        date_value = format_datetime(datetime.datetime.today(), 'MMMM d, y', tzinfo=timezone, locale=locale)
+    utc_zone = get_timezone('UTC')
+    to_zone = get_timezone(timezone)
+
```

Comment:
There's nothing "different" about these two blocks any more, so let's kill the arbitrary newline.